core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -785,6 +785,10 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
   </div>
   <style>
 
+    body {
```

Comment:
@NishealJ, isn't this a very big change? Please add this property to the specific dom.

Reply:
> Also, the scroll was not working while dragging an exploration to the last position because of
overflow-x : hidden which has been flipped to visible.

How is this related to "x" (As the issue is with vertcal scrolling)? can you please explain?

Reply:
Hi, from the oppia.css the default overflow-x is hidden in the body which is to be overwritten in learner_dashboard.html page.

Also, yes the issue was in vertical scroll but it fixed by overflow-x. I too found it a bit strange. i’ll make a debug report and share it to you here :)

While debugging i found overflow-x : hidden responsible for this unexpected behaviour, after flipping it works perfect.

Reply:
Hi, I've shared a report [here](https://docs.google.com/document/d/1Kea7CPHdQYLVrzBnt7yzSC1Um1n6oyNSHUOL5MJdhQc/edit?usp=sharing). Thanks !

Reply:
Can we scope this change to just the learner dashboard? Otherwise it will affect body tags all over..

Reply:
Hi @nithusha21,
I've scoped this to learner_dashboard.html, as an internal CSS.
This will affect only the learner dashboard.

Reply:
Oh ok. Thanks for clarifying.